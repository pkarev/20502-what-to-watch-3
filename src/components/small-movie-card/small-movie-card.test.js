import React from 'react';
import renderer from 'react-test-renderer';
import SmallMovieCard from './small-movie-card';

const movie = {
  id: 1,
  name: `Fantastic Beasts`,
  posterSmall: `/img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
  trailer: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`
};

it(`Render SmallMovieCard`, () => {
  const tree = renderer
    .create(<SmallMovieCard movie={movie} onCardClick={() => {}}/>, {
      createNodeMock: () => ({})
    })
    .toJSON();

  expect(tree).toMatchSnapshot();
});
