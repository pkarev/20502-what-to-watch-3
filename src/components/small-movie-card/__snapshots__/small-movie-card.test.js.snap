// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render SmallMovieCard 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onClick={[Function]}
>
  <video
    className="small-movie-card__image"
    controls={false}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    poster="/img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
    style={
      Object {
        "objectFit": "cover",
        "width": "100%",
      }
    }
  >
    <source />
  </video>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
      href="movie-page.html"
    >
      Fantastic Beasts
    </a>
  </h3>
</article>
`;
